src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return {
+        count: state.count + action.payload
```

Comment:
If you deleted the payload in your actions, you would replace this so: `count: state.count + 1`. Again both work, but this is another way to do it